Svg/Platform/SkiaTextRenderer.cs

```diff
@@ -76,32 +72,35 @@ private void RenderFill(SvgTextBase txt, ISvgRenderer renderer)
                         fillOpacity);
 
                     pen.Paint.IsStroke = false;
+                    DrawWithPen(txt, renderer, pen);
+                }
+            }
+        }
 
-                    if (txt.Parent is SvgTextBase parent && txt is SvgTextSpan)
-                    {
-                        var x = parent.X.FirstOrDefault().Value + txt.Dx.FirstOrDefault().Value;
-                        var y = parent.Y.FirstOrDefault().Value + txt.Dy.FirstOrDefault().Value;
-
-                        foreach (var svgElement in parent.Children)
-                        {
-                            var child = (SvgTextSpan)svgElement;
-                            if (txt.Equals(child))
-                            {
-                                break;
-                            }
+        private void DrawWithPen(SvgTextBase txt, ISvgRenderer renderer, SkiaPen pen)
+        {
+            if (txt.Parent is SvgTextBase parent && txt is SvgTextSpan)
+            {
+                var x = parent.X.FirstOrDefault().Value + txt.Dx.FirstOrDefault().Value;
```

Comment:
I am sure this is not nearly enough.
Since a svgText can hav _multiple_ tspan elements as children, the `dx` and `dy` values are  the relative delta to the _previous tspan_, __not__ the parent!
See here:
![image](https://user-images.githubusercontent.com/1175751/220364949-41e4b0e1-4d11-42dd-aa52-3c2dd77d68fe.png)

Please do not blindly play around but figure out what each attribute means and how to implement.
Didn't we already talk about using RenderChildren, because that already takes transformations into account?